completion/available/fabric.completion.bash

```diff
@@ -44,6 +44,7 @@ case "$OSTYPE" in
 		__FAB_COMPLETION_MTIME_COMMAND="stat -f '%m'"
 		;;
 	*)
+		# shellcheck disable=SC2089
 		__FAB_COMPLETION_MTIME_COMMAND="stat -c '%Y'"
```

Comment:
In fact, this will cause a problem. `$(__fab_chache_mtime)` on line 118 will result in `'<number>'` instead of `<number>`, and then the comparison with `-gt` is broken. An easy fix would be to remove the single quotes:
```suggestion
		# shellcheck disable=SC2089
		__FAB_COMPLETION_MTIME_COMMAND="stat -c %Y"
```

but you could actually follow ShellCheck's suggestion as

```suggestion
		__FAB_COMPLETION_MTIME_COMMAND=(stat -c '%Y')
```

so that the later call of `${__FAB_COMPLETION_MTIME_COMMAND}` can be written in the form of `"${__FAB_COMPLETION_MTIME_COMMAND[@]}"`.

Reply:
I have considered that but I was not sure this wouldn't get rid of the quotes. ok, I'll do it that way.

Reply:
@hyperupcall solves this with the latter way in #2295.